### PR TITLE
WIP: Feature blockchain create new organizations and accounts

### PR DESCRIPTION
**Pull request by gangadharjannu**
Adds express generator folder structure for nodejs.
Improves front end connection to backend

### CODE REVIEW

File: back-end/routes/new-accounts.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
+var express = require('express');
+const fetch = require('node-fetch');
+const settings = require('../settings');
+const mongoose = require('mongoose');
+var router = express.Router();
+
+var accountsSchema = new mongoose.Schema(
```
Comment (theblockstalk) [inline back-end/routes/new-accounts.js:7]: should we move this to a separate file for collection schemas?

File: back-end/routes/index.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+var express = require('express');
```
Comment (theblockstalk) [inline back-end/routes/index.js:1]: delete route

File: front-end/package.json [hunk 3]
```diff
@@ -3,6 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "type": "module",
+  "proxy": "http://localhost:4000",
```
[the remark this answers is not in the document]
Comment (gangadharjannu) [reply front-end/package.json:6]: This is only for dev environment and we can't set the dynamic data in package.json

File: back-end/routes/new-accounts.js (hunk 1 above)
Comment (gangadharjannu) [reply back-end/routes/new-accounts.js:7]: I already have setup a new file for schema in another folder. This is only for testing purpose.

**Pull request merged**